lib/src/commands/Commands.ts

```diff
@@ -86,11 +87,12 @@ export class Commands {
     const input = _.cloneDeep(simpleApi);
 
     const layout = this.layoutTreeParser.parse(input);
+    const copiedLayout = _.cloneDeep(layout);
     this.layoutTreeCrawler.crawl(layout);
 
     const commandId = this.uniqueIdProvider.generate('push');
     const result = this.nativeCommandsSender.push(commandId, componentId, layout);
-    this.commandsObserver.notify('push', { commandId, componentId, layout });
+    this.commandsObserver.notify('push', { commandId, componentId, copiedLayout });
```

Comment:
@jfrolich This line broke the tests, it should probably be `layout: copiedLayout`